txnkv/transaction/txn.go

```diff
@@ -1700,6 +1700,10 @@ func (txn *KVTxn) StartTS() uint64 {
 	return txn.startTS
 }
 
+func (txn *KVTxn) CommitTS() uint64 {
```

Comment:
There are still some tests using `txn.GetCommitTS` in `1pc_test.go`.

```bash
cd integration_tests && go test ./...
```

Reply:
@you06 Seems to be an enhancement to add a check before and after the  `ts allocation`, how about filing an issue for it?

Reply:
The integration_tests are fixed.

Sounds a good idea to check store's last_commit_ts is smaller than start_ts, because store's last_commit_ts is not directly allocated from pd.

Reply:
> how about filing an issue for it?

@cfzjywxk  It's mentioned in pingcap/tidb#57165 IMO.

> In theory, all read requests occurring after the write transaction should use a `read_ts` greater than the previous known transaction's `commit_ts`.